Samples/BruTile.Demo/MainWindow.xaml.cs

```diff
@@ -14,7 +14,7 @@ public MainWindow()
     {
         InitializeComponent();
 
-        foreach (var knownTileSource in Enum.GetValues(typeof(KnownTileSource)).Cast<KnownTileSource>())
+        foreach (var knownTileSource in GetKnownTileSourceValues())

```

Comment:
This was changed because the previous method did not work properly because of reflection in AOT?

Reply:
The Microsoft.XmlSerializer.Generator removes the Reflection problem But the AOT merging does not include the generated XML Deserialization assembly so it doesn't work there.
see
https://gist.github.com/filipnavara/1e8831c256498bad53b3aae94af87a20

But this requires some msbuild targets, maybe causing some build problems later.

I guess I will go for the explicit xml deserialization approach, generating the code with ai and correcting it like here

https://github.com/DotSpatial/DotSpatial/pull/1500